includes/Tracking.php

```diff
@@ -104,6 +104,7 @@ public function get_settings() {
 		return [
 			'trackingAllowed' => $this->is_active(),
 			'trackingId'      => self::TRACKING_ID,
+			'appVersion'      => WEBSTORIES_VERSION,
```

Comment:
Is the name of the key a little confusing. Is plugin version or webstories version.

Reply:
`app_version` is what GA uses, so I went with `appVersion` for consistency. Given the context I think it should be clear enough that this is the Web Stories plugin vesion.

Reply:
So I will stay with `appVersion` for now